engine/components/renderables/AnimationRenderable.hpp

```diff
@@ -0,0 +1,31 @@
+//
+// Created by 11896 on 19/11/2024.
+//
+
+#ifndef BLOCKYENGINE_ENGINE_COMPONENTS_RENDERABLES_ANIMATIONRENDERABLE_HPP_
+#define BLOCKYENGINE_ENGINE_COMPONENTS_RENDERABLES_ANIMATIONRENDERABLE_HPP_
+#include <string>
+#include <vector>
+#include <iostream>
+#include <glm/vec4.hpp>
+#include "SpriteRenderable.hpp"
+
+class AnimationRenderable : public SpriteRenderable {
```

Comment:
AnimationRenderable is in principe niet nodig: doe de loadFrames enzo in de AnimationController en dan kan je hetzelfde met de SpriteRenderable. het enige wat de SpriteRenderable dan bij hoeft te houden is de glm::ivec4 sourceRect{}. standaard, van SpriteRenderable z'n contructor, is deze misschien nullptr of iets van een andere default waarde. de AnimationController haakt dan in op een SpriteRenderable